### PR TITLE
OEL-750: BCL Upgrade to 0.14.0

### PR DESCRIPTION
**Pull request by escuriola**
Jira issue(s):
- https://citnet.tech.ec.europa.eu/CITnet/jira/browse/OEL-750

Includes: 
- Header: https://citnet.tech.ec.europa.eu/CITnet/jira/browse/OEL-756
- Links: https://citnet.tech.ec.europa.eu/CITnet/jira/browse/OEL-761

### CODE REVIEW

**Review by GilNovacomm — DISMISSED** [review]
Approved.

File: templates/patterns/header/pattern-header--variant-ec.html.twig [hunk 1]
```diff
@@ -1,18 +1,11 @@
...
-    color_set: 'light',
-    collapse_id: 'navbarNavDropdown',
-    attributes: attributes
-  }) }}
-{% endblock %}
+{% extends "pattern-header.html.twig" %}
+{%  block header %}
+  {% set _header_attributes = create_attribute().addClass(['ec__header', 'bcl-header']) %}
+<header {{ _header_attributes }}>
+  {% endblock %}
```
Comment (drishu) [inline templates/patterns/header/pattern-header--variant-ec.html.twig]: typo here, to much space
```suggestion
{% endblock %}
```

File: templates/patterns/header/pattern-header--variant-ec.html.twig [hunk 2]
```diff
@@ -1,18 +1,11 @@
...
-  {{ pattern('navbar', {
-    navbar_left: header_left,
-    navbar_right: header_right,
-    color_set: 'light',
-    collapse_id: 'navbarNavDropdown',
-    attributes: attributes
-  }) }}
-{% endblock %}
+{% extends "pattern-header.html.twig" %}
+{%  block header %}
```
Comment (drishu) [inline templates/patterns/header/pattern-header--variant-ec.html.twig]:
```suggestion
{% block header %}
```

File: templates/patterns/header/pattern-header--variant-eu.html.twig [hunk 3]
```diff
@@ -1,16 +1,11 @@
...
-{% block content %}
-  {{ pattern('navbar', {
-    navbar_left: header_left,
-    navbar_right: header_right,
-    color_set: 'dark',
-    collapse_id: 'navbarNavDropdown',
-    attributes: attributes
-  }) }}
+{% extends "pattern-header.html.twig" %}
+{%  block header %}
```
Comment (drishu) [inline templates/patterns/header/pattern-header--variant-eu.html.twig]:
```suggestion
{% block header %}
```

**Review by drishu — CHANGES_REQUESTED** [review]

**Review by drishu — APPROVED** [review]

**Review by GilNovacomm — APPROVED** [review]
Approved

**Pull request merged**